specs/validator/tableland-openapi-spec.yaml

```diff
@@ -181,7 +192,11 @@ paths:
         "400":
           description: Invalid chain or table identifier
         "404":
-          description: Table not found
+          description: Table Not Found
+        "429":
+          description: Too Many Requests
+        "500":
```

Comment:
This is the only one where I've encountered an actual 500 with, so just putting here to be explicit.

Reply:
Any API could also return a status code 500. Generally, a status code 500 is returned in any uncontrolled error when processing the API. For example, if when executing a database read fails for whatever reason, that will mean a 500.

This is a similar "general" rule as with 429. I wonder if they should be part of the formal description or 429/500 are general enough not to be included in an API spec. 

In any case, including them looks fine to me to make people more aware. But I believe most API clients should usually be aware of 429s or 500s.